tests/smoke/events.js

```diff
@@ -81,11 +83,24 @@
 					shiftKey: false
 				}));
 
-				// wait for spellcheck to fire after the spacebar
+				resumeAfter(editor, 'spellCheckComplete', function () {
```

Comment:
I think YUI has promises, but resumeAfter (which is a ckeditor/bender util function to play nicer with editor events) seems to only support callbacks.

Naming the callbacks makes sense though. Sometimes end up just nesting things to play out an idea and never really fix it afterwards.